docs/gpa.md

```diff
@@ -180,25 +179,30 @@ This requires the [Covered By](https://github.com/open-contracting-extensions/oc
         <td>the subject-matter of the procurement;</td>
         <td markdown=1>
 
-* 
+* Enter it in `tender/description`
+* If `tender/description` is not empty, enter it after the existing content
```

Comment:
Elsewhere we use the construction "Add this to" if appending.

Reply:
I've finally discarded the whole article (https://github.com/open-contracting-extensions/government-procurement-agreement/pull/24#discussion_r428798827)